### ISSUE TITLE
LAVFilters-0.74.1-20 Dolby Atmos issue again

### ISSUE THREAD

**Issue by cmichaelhk** [Nevcairiel/LAVFilters#302]
2 months before, installed the nightly build 0.74.1-20(bitstream to av receiver), fixed the dobly atmos problem(no sound after few times seek, receiver keep displaying Dobly Atmos) with UHD Black Hawk Down. But today, when I watch uhd Alita Battle Angel(rip) and uhd Avengers Endgame(iso rip) same problems found again. After that I try Black Hawk Down(rip and original disc with anydvd) agian and it also got no sound after few times seek.

I already tried roll-back the RX480 driver to 19.4.3-apr22 and 19.7.1-july7 still no luck...….

**Comment by Nevcairiel** [comment]
If LAV didn't change in 2 months, and the issue still came back, its probably something else, and not LAV.

**Comment by cmichaelhk** [comment]
Yes I know. So I am now reinstall windows and will test again....

**Comment by cmichaelhk** [comment]
after install latest windows 10, problem are same :(

**Comment by cmichaelhk** [comment]
after re-install latest Windows 10 and only install rx480 driver, Lav filter, MPC-BE, same no sound problem found. Will it caused by windows update? As I found the problem in pervious installed windows seem occure after windows updated.

**Comment by cmichaelhk** [comment]
I am sorry that I seem found out the problems.
As the yamaha CX-A5100 default 4k mode is mode 2(just like safe mode in pc) that will cause HDMI in low bandwidth so that I think that caused the sound problem when playing 4k hdr content.

How to know you are in low bandwidth mode for me in windows 10? Goto the page that change screen resolution page. If the recommend resolution is 1920x1080 that is in mode 2.
To change back to 4k native mode, please visit
https://www.google.com/amp/s/www.avsforum.com/forum/90-receivers-amps-processors/2963648-how-fix-problem-yamaha-cx-a5100-other-yamaha-avrs-don-t-pass-hdr.html%3famp=1

This fucking setting already cause me confused when I just changed to 4k TV for a years.

P.S. If you got blank screen problem when switch on the PC after change to mode 1, try enable CSM support in BIOS (when I disabled the CSM, my PC with rx480 will not start + blank screen + no beep sound)

**Comment by Nevcairiel** [comment]
So just to be clear,  with it set into proper 4K mode, Atmos now also functions properly?

**Comment by cmichaelhk** [comment]
Yesterday I test the atmos sound seem ok.....but today sound will lost again.....

**Comment by danbezerra** [comment]
Atmos has been working flawless to me as well since the 0.74 update. But yesterday I watched Toy Story 4 (always a Disney title!) and first I could sense a micro drop of sound during what seems to be seamless branching points. That was very minor until the very end of the movie where the sound just dropped completely. I had to go back one chapter to get the sound going on again and then skip forward to where I was before. So it seems that this issue is not yet fully addressed, although much much better than pre 0.74. as FYI, I am using a Denon 3200.

**Comment by aoaaron** [comment]
Same .. still ongoing issue.

**Comment by ParcelRot** [comment]
I found that if I enabled MSI interrupts for the NVIDIA drivers most playback issues disappear. I wonder if that is an option for you guys (even for AMD)?

**Comment by ParcelRot** [comment]
Oh another thing - remuxing Atmos titles (particularly Disney ones) into MKV could result in dropouts if the remuxing tool is not careful with how the Atmos packets are written. The discussion on this is pretty involved and out of scope here though.
